specs/transport/car/carv2/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "CARv2 Specification"
+eleventyNavigation:
+  synopsys: "Specification for the CARv2 format"
+---
+
+# Specification: Content Addressable aRchives (CAR / .car) v2
+
+**Status: Draft**
+
+* [Summary](#summary)
+* [Format Description](#format-description)
+  * [Pragma](#pragma)
+  * [Header](#header)
+  * [Characteristics](#characteristics)
+  * [CARv1 data payload](#carv1-data-payload)
+  * [Index payload](#index-payload)
+  * [Index format](#index-format)
+    * [Format `0x300000`: IndexSorted](#format-0x300000-indexsorted)
+* [Implementations](#implementations)
+* [Test Fixtures](#test-fixtures)
+
+## Summary
+
+CARv2 is a minimal upgrade to the [CARv1](../carv1/index.md) format with the primary aim of adding an optional index within the format for fast random-access to blocks.
+
+CARv2 makes use of CARv1 by wrapping a properly formed CARv1 with a prefix containing a pragma and header, and a suffix containing the optional index data. Once the offset and length of the CARv1 bytes are determined using CARv2 parsing rules, an existing CARv1 decoder can be used to read the roots and `CID:Bytes` pairs. Likewise, a CARv1 encoder can be used to encode this data for wrapping by a CARv2 encoder.
+
+## Format Description
+
+CARv2 consists of:
+
+1. An 11-byte pragma (or sequence of "magic bytes") that identify the data as a CARv2 format.
+2. A header describing data payload offset and length and index offset within the container as well as a bitfield for setting various capabilities supported by the container.
```

Comment:
should we list all that the header contains here? IMO it would be easier to simply refer to the Header section below.